File: sub-web-model/src/main/java/org/github/base/mapper/ITagMapper.java
package org.github.base.mapper;

import org.github.base.entity.TagEntity;
import org.github.base.IMapper;
import org.github.mybatis.MyBatisMapper;

/**
 * <p>
 *  Mapper 接口
 * </p>
 *
 * @author JYD_XL
 * @since 2021-05-08
 */
@MyBatisMapper
public interface ITagMapper extends IMapper<TagEntity> {}

File: sub-mysql-blog-service/src/main/java/org/github/mysql/blog/base/mapper/ITagMapper.kt
@file:Suppress("RedundantSemicolon")

package org.github.mysql.blog.base.mapper;

import org.github.mysql.blog.base.entity.TagEntity;
import org.github.base.IMapper;
import org.github.mybatis.MyBatisMapper;

/**
 *  Mapper 接口
 *
 * @author JYD_XL
 */
@MyBatisMapper
interface ITagMapper : IMapper<TagEntity>
